src/test/java/org/javawebstack/orm/test/UpdateOnlyIfIsDirtyTest.java

```diff
@@ -0,0 +1,53 @@
+package org.javawebstack.orm.test;
+
+import org.javawebstack.orm.ORM;
+import org.javawebstack.orm.ORMConfig;
+import org.javawebstack.orm.exception.ORMConfigurationException;
+import org.javawebstack.orm.test.shared.models.JustString;
+import org.javawebstack.orm.wrapper.QueryLogger;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class UpdateOnlyIfIsDirtyTest extends ORMTestCase {
+
+    @Test
+    public void testOnlyUpdateIfIsDirty() throws ORMConfigurationException {
+        boolean[] updated = new boolean[1];
```

Comment:
I would maybe add a short reason as to why we do this
```suggestion
        // Using an array to make force mutability inside lamda expression
        boolean[] updated = new boolean[1];
```

Reply:
Except if this is a common thing in the java word. In this case feel free to merge.